src/client/routes/vessels-router.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { Route, withRouter, Link } from "react-router-dom";
+
+import VesselDetails from "./../components/VesselDetails";
+
+const VesselRouter = props => {
+  return (
+    <React.Fragment>
+      <div className="container">
+        <Route exact path="/vessels" component={VesselDetails} />
```

Comment:
Should be `/` but we also discussed that on Slack.

Reply:
removed because not related to this task